mep/people/templates/people/snippets/manifest_license.html

```diff
@@ -0,0 +1,16 @@
+{# display & link IIIF manifest license and provider logo #}
+{% load static mep_tags %}
+<div class="rights-statement">
+    {# logo for IIIF provider of digitized item #}
+    {% if manifest.logo %}
+        {# NOTE: could link to findingaids version via rendering url in extra data #}
+        <img class="logo" src="{{ manifest.logo }}"/>
```

Comment:
it's annoying that there's not a good candidate for an `alt=` for this. i was looking at maybe [`attribution`](https://iiif.io/api/presentation/2.1/#attribution) as the closest one, but it's optional and you can provide multiple. is there a sane default we could use, or should we just hide this from SRs entirely? feels wrong because it's separate info from the rights statement (and potentially important), but not sure what else to do.

Reply:
I agree, I was looking again at the manifest because it feels weird to have a logo with no other information - it isn't really informative if you don't already recognize the logo. 

We might want to hide it from the screen readers for now, but feel free to ask the library devs about adding an attribution text or asking for a better alternative. IIRC they only added the logo when I asked them to do so for the derrida project (but they did do it quickly).

What do you think about using the logo to use the rendering logo to link to the rendering url from the manifest, which for this content would go to the finding aid version of the item?  (not right away but in future)